FitTracker/app/src/main/java/com/pps/globant/fittracker/MainActivity.java

```diff
@@ -46,9 +36,10 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     @Override
-    protected void onStop() {
-        super.onStop();
-        presenter.fbLogOut();
+    protected void onDestroy() {
+        super.onDestroy();
+        BusProvider.unregister(presenter, presenter.getModel());
```

Comment:
You should move this into presenter, and call it since onPause() activity method